mitodl/micromasters
# Added timestamp fields to ExamProfile

#### What are the relevant tickets?
#3025 

#### What's this PR do?
This adds `updated_on` and `created_on` fields to `ExamProfile`. We need this because we previously sent the value of `Profile.updated_on` when syncing `ExamProfile` records and we need to be able to update that without affecting the user's `Profile`.

#### How should this be manually tested?
- Create an `ExamProfile` for your user
- Run the command `./manage.py resync_exam_profiles --ids=678,567,567`, which shouldn't match any records
- Run the command `./manage.py resync_exam_profiles --ids=?`, passing your `ExamProfile.id` which shouldn't match the one record
- Run the command `./manage.py resync_exam_profiles`, which should match all records

#### Where should the reviewer start?
See exams/models.py, but bulk of changes are to exams/pearson/writers.py
